pkgs/applications/window-managers/i3/lock-color.nix

```diff
@@ -1,18 +1,18 @@
-{ stdenv, fetchFromGitHub, which, pkgconfig, libxcb, xcbutilkeysyms
+{ stdenv, fetchurl, which, pkgconfig, libxcb, xcbutilkeysyms
 , xcbutilimage, pam, libX11, libev, cairo, libxkbcommon, libxkbfile }:
 
 stdenv.mkDerivation rec {
-  rev = "c8e1aece7301c3c6481bf2f695734f8d273f252e";
-  version = "2.7-2016-09-17";
+  version = "2.8";
   name = "i3lock-color-${version}";
-  src = fetchFromGitHub {
```

Comment:
Does this not turn this whole thing into a different piece of software?
Now you pull `i3lock` without the colour extensions from `i3lock-color`.